### PR TITLE
Save-order-of-songs-into-firebase-after-drag-and-drop

### PR DESCRIPTION
**Pull request by gamesminer**

### CODE REVIEW

File: src/components/MySongs/MySongs.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ export class MySongsComponent {
 
   mountChildren() {
     this.table = new SongsTableComponent(this.tableContainer, {
+      ondrag: true,
```
Comment (ihor4568) [inline src/components/MySongs/MySongs.js]: onDrag

File: src/components/SongsTable/SongsTable.js [hunk 2]
```diff
@@ -130,15 +132,22 @@ export class SongsTableComponent {
       `.songs-table__th-icon_${this.state.columnName}`
     );
     this.tableBody = this.mountPoint.querySelector(".songs-table__body");
+    this.tableRow = this.mountPoint.querySelectorAll(".songs-table__row");
```
Comment (ihor4568) [inline src/components/SongsTable/SongsTable.js]: tableRows -- use plural if it is array

File: src/components/SongsTable/SongsTable.js [hunk 3]
```diff
@@ -161,6 +170,16 @@ export class SongsTableComponent {
     this.dragElement = e.target;
   }
 
+  handleDrop(e) {
+    e.preventDefault();
+    this.querySelectors();
+    const songsTable = [];
+    Array.from(this.tableRow).forEach(item => {
```
Comment (ihor4568) [inline src/components/SongsTable/SongsTable.js]: please use `map` instead

File: src/services/MusicService.js [hunk 4]
```diff
@@ -77,4 +77,14 @@ export class MusicService {
       )
     );
   }
+
+  static rewriteMySong(arr) {
```
Comment (ihor4568) [inline src/services/MusicService.js]: please think of more descriptive name

**Review by OlegShynkarenko — APPROVED** [review]

**Pull request merged**